### PR TITLE
bugfix[codecov]: revert container version

### PR DESCRIPTION
**Pull request by sebastianpfischer**
I reverted the code in #266 that broke the codecov github job

### CODE REVIEW

**Comment by Pog3k** [comment]
I changed this because our unittest  will fail with the "ubuntu" container. 
The unittest creates some test files with different timestamps in the meta file information.
In python we create a list of files which should actually be sorted by date but it doesn't work with the ubuntu image. 
Somehow it cant handle the date.
In wsl it works with an ubuntu.

**Review by Pog3k — CHANGES_REQUESTED** [review]
maybe try other lightweight image .. debian ?

**Comment by sebastianpfischer** [comment]
> maybe try other lightweight image .. debian ?

@Pog3k , I think we forgot to read the documentation -> https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners

Our only possibilities are:
* `windows-latest` or `windows-2022`, `windows-2019`
* ` ubuntu-latest` or `ubuntu-22.04`
* `ubuntu-20.04` (with less runners)
* `macos-13` or `macos-13-xl`, `macos-latest`, `macos-12`, `macos-latest-xl` or `macos-12-xl`, `macos-11``

So only Ubunto-22.04 is here a possibility for us.
Do I forget something?

**Comment by Pog3k** [comment]
Testet it on my pc with ubuntu:latest and python3.10 and it works.. can your try to use 3.10 instead of 3.7

File: .github/workflows/codecov.yml [hunk 1]
```diff
@@ -18,13 +18,22 @@ jobs:
...
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
+      # Pre requisite installation
+      - name: Prerequisite installation
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
```
Comment (Pog3k) [inline .github/workflows/codecov.yml:32]: `python-version: '3.10`

**Comment by sebastianpfischer** [comment]
@Pog3k, did you also test it with 3.8?

File: .github/workflows/codecov.yml (hunk 1 above)
Comment (sebastianpfischer) [reply .github/workflows/codecov.yml]: Yea ok, I probably need to maintain the compatibility with 3.7 via the Eclipse Jenkins right?

**Comment by Pog3k** [comment]
i updated it and codecov seems now working. 

So some weird behaviour with ubuntu and the old python 3.7 which is actually discontinued for the latest ubuntu...

**Review by Pog3k — APPROVED** [review]

**Review by flo117 — APPROVED** [review]

**Comment by sebastianpfischer** [comment]
@Pog3k , thanks, have you seen: https://app.codecov.io/gh/eclipse/kiso-testing/commit/96b1b4703386b6b7a2a98bedf1f156c6df4f859a?

**Pull request merged**